Microsoft.Toolkit.Uwp.UI.Controls.Input/RichSuggestBox/RichSuggestBox.xaml

```diff
@@ -1,8 +1,5 @@
 <ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                    xmlns:contract7NotPresent="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract,7)"
```

Comment:
Missed this before @huynhsontung I believe contract 7 is 1809 which is our min version, so we should be able to assume that stuff is there?

Reply:
Turns out the control doesn't need it anyway. Removed 👍